src/engine/config.rs

```diff
@@ -50,7 +50,7 @@ pub struct Configuration {
     ///
     /// [`Type`]: struct.Type.html
     #[serde(default)]
-    model: Vec<Type>,
+    pub model: Vec<Type>,
```

Comment:
Made the model public to allow modification in the before_engine_build event hook.